HW03/Ekaterina_Vashkevich/student.rb

```diff
@@ -0,0 +1,33 @@
+# class describes the state and behavior of students and
+# interaction with students
+class Student
+  attr_reader :name, :surname, :homeworks, :mentors, :notifications
+
+  def initialize(name, surname)
+    @name = name
+    @surname = surname
+    @homeworks = []
+    @mentors = []
+    @notifications = []
+  end
+
+  def sumbit_homework!(homework)
+    @homeworks << homework
+    notify_homeworks(homework)
```

Comment:
What do you think? 🤔 
```suggestion
    notify_mentors(homework)
```

Reply:
so more clear) thanks)